.github/workflows/build.yml

```diff
@@ -1,186 +1,186 @@
+        # packages, and so `yarn build` should always run. This `if` check is therefore only there for testing CI issues
+        # where the built packages are beside the point. In that case, you can change `BUILD_CACHE_KEY` (at the top of
+        # this file) to a constant and skip rebuilding all of the packages each time CI runs.
+        if: steps.cache_built_packages.outputs.cache-hit == ''
+        run: yarn build
         # yarn.lock cannot be dirty when releasing a new version.
-       - name: Check if yarn.lock is dirty
-         if: steps.cache_built_packages.outputs.cache-hit == ''
-         run: yarn install --frozen-lockfile
+      - name: Check if yarn.lock is dirty
+        if: steps.cache_built_packages.outputs.cache-hit == ''
+        run: yarn install --frozen-lockfile
 
-     outputs:
-       # this needs to be passed on, because the `needs` context only looks at direct ancestors (so steps which depend on
-       # `job_build` can't see `job_install_deps` and what it returned)
-       dependency_cache_key: ${{ needs.job_install_deps.outputs.dependency_cache_key }}
+    outputs:
+      # this needs to be passed on, because the `needs` context only looks at direct ancestors (so steps which depend on
+      # `job_build` can't see `job_install_deps` and what it returned)
+      dependency_cache_key: ${{ needs.job_install_deps.outputs.dependency_cache_key }}
 
-   job_carthage_build:
-     name: Build Carthage Dependencies
-     runs-on: macos-latest
-     timeout-minutes: 15
-     steps:
-       - name: Check out current commit (${{ github.sha }})
-         uses: actions/checkout@v2
-       - name: Check Carthage build cache
-         uses: actions/cache@v2
-         id: cache_built_carthage
-         with:
-           path: ${{ env.CACHED_CARTHAGE_PATHS }}
-           key: ${{ hashFiles('src/ios/Cartfile') }}
-       - name: Install Carthage
-         if: steps.cache_built_carthage.outputs.cache-hit == ''
-         run: brew install carthage
-       - name: Build Cocoa SDK from Carthage
-         if: steps.cache_built_carthage.outputs.cache-hit == ''
-         env:
-            GITHUB_ACCESS_TOKEN: ${{ secrets.GH_RELEASE_PAT }}
-         run: make build
+  job_carthage_build:
+    name: Build Carthage Dependencies
+    runs-on: macos-latest
+    timeout-minutes: 30
+    steps:
+      - name: Check out current commit (${{ github.sha }})
+        uses: actions/checkout@v2
+      - name: Check Carthage build cache
+        uses: actions/cache@v2
+        id: cache_built_carthage
+        with:
+          path: ${{ env.CACHED_CARTHAGE_PATHS }}
+          key: ${{ hashFiles('src/ios/Cartfile') }}
+      - name: Install Carthage
+        if: steps.cache_built_carthage.outputs.cache-hit == ''
+        run: brew install carthage
+      - name: Build Cocoa SDK from Carthage
+        if: steps.cache_built_carthage.outputs.cache-hit == ''
+        env:
+          GITHUB_TOKEN: ${{ secrets.GH_RELEASE_PAT }}
```

Comment:
fixes 
```
fatal: unable to access 'https://github.com/getsentry/sentry-cocoa.git/': URL using bad/illegal format or missing URL
```
Cordova was the only project using GITHUB_ACCESS_TOKEN where all other projects sets GITHUB_TOKEN